beacon-chain/node/node.go

```diff
@@ -233,7 +233,7 @@ func (b *BeaconNode) registerBlockchainService(ctx *cli.Context) error {
 }
 
 func (b *BeaconNode) registerOperationService() error {
-	var p2pService *p2p.Server
+	var p2pService *hobbits.HobbitsNode
 	if err := b.services.FetchService(&p2pService); err != nil {
```

Comment:
We need to write some `b.services.FetchP2PService()` which can return the interface backed by either p2p or hobbits. 

Let's sync on this idea soon because the changes in this file are not going to work without the `--hobbits` flag.

Reply:
+1